### DISCUSSION TITLE
Redundancy

### DISCUSSION THREAD

**Discussion by gius** [Eventuous/eventuous#324] [Q&A]
How would you configure Eventuous-based application for redundancy/HA or even for scaling purposes?
I am concerned about running multiple application instances and their event handlers / subscriptions being called on each instance, particularly when using PostgreSQL event store.

Thank you for any ideas!

**Comment by fbjerggaard** [comment]
There has been some discussion in #162 regarding this.
Right now there is no way to run Eventuous distributed - it really needs to only be ran in a single instance
